### DISCUSSION TITLE
Merging

### DISCUSSION THREAD

**Discussion by JimmyTron** [SpaceyaTech/Team-Rio-Django#6] [General]
#Merging restrictions
we will restrict merging to the main branches until certain checks have passed and agreed upon since this is not a one person show and we will need members to contribute to the progress of this team.

**Comment by LionelMv** [comment]
Anyone who can assist me to understand why we need to fork a repo. And what is the difference between that and cloning the repo directly, creating a branch and pushing it as a branch and then creating a pull request
